lib/rubocop/cop/metrics/block_length.rb

```diff
@@ -13,12 +13,28 @@ class BlockLength < Cop
         LABEL = 'Block'.freeze
 
         def on_block(node)
-          return if excluded_methods.include?(node.send_node.source)
+          return if excluded_method?(node)
           check_code_length(node)
         end
 
         private
 
+        def excluded_method?(node)
+          node_receiver = node.receiver && node.receiver.source.gsub(/\s+/, '')
+          node_method = String(node.method_name)
+
+          excluded_methods.any? do |config|
+            receiver, method = config.split(/\.(\w+$)/)
```

Comment:
I guess we can just use `/\./`.

Reply:
Sure thing. The intention behind this regex was to support excluding of method chains, e.g.

```yaml
Metrics/BlockLength:
  Max: 2
  ExcludedMethods: ['cool.method.chain']
```

```ruby
cool.method.chain do
  # ...
end
```
Anyway, I find this idea quite questionable, so fixed.